### PR TITLE
add when nedomi was started and what version it is to the status

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

File: handler/status/status_page.go [hunk 1]
```diff
@@ -95,9 +98,29 @@ type statisticsRoot struct {
 	Responded     uint64           `json:"responded"`
 	NotConfigured uint64           `json:"not_configured"`
 	InFlight      uint64           `json:"in_flight"`
+	Version       version          `json:"version"`
+	Started       time.Time        `json:"started"`
 	CacheZones    []zoneStatistics `json:"zones"`
 }
 
+type version struct {
```
Comment (na--) [inline handler/status/status_page.go]: Why is this needed? Why not use types.AppVersion directly?

Comment (MStoykov) [inline handler/status/status_page.go]: Because it doesn't have json tags and adding json tags, because a handler needs them, doesn't seem like the correct way of doing it. 

Also this way it's only the status handler that defines how and what it returns not a type in another package. 

Btw types.AppStats is similar.

Comment (na--) [inline handler/status/status_page.go]: :+1:
